docs/README.md

```diff
@@ -3,3 +3,4 @@
 - [Get Started](./get-started.md)
 - [Using in test tools](./using-test-tools.md)
 - [Workflow Syntax](./workflow-syntax.md)
+- [CLI Reference](./cli-reference.md)
```

Comment:
I think we need to reorder this bit

1. Get Started
2. CLI Reference
3. Workflow Syntax
4. Using in Test Tools